keras/models.py

```diff
@@ -3,7 +3,19 @@
 import theano
 import theano.tensor as T
 import numpy as np
-import warnings, time, copy
+import warnings, time, copy, yaml
+
+from .layers.convolutional import *
```

Comment:
Let's avoid `*` imports, even if the import list is very long. The namespace in `models` is very busy, and we need what is free and what isn't.